### PR TITLE
Added Batch proposals support

### PR DESCRIPTION
**Pull request by ernestognw**
## Description

Added docs for batches proposal type

### CODE REVIEW

**Review by pasevin — APPROVED** [review]

File: docs/modules/ROOT/pages/admin.adoc [hunk 1]
```diff
@@ -85,6 +85,23 @@ Custom actions can also be _repeated_, which will present you to a pre-filled fo
 
 WARNING: Certain ABI types, such as nested structs, are not yet supported. Contact us if you need to call a function that is currently unsupported!
 
+[[batches]]
+=== Batches
+
+A _batch proposal_ is a call to multiple functions from different contracts. It allows you to atomically execute multiple actions in a single transaction.
```
Comment (shahnami) [inline docs/modules/ROOT/pages/admin.adoc:91]: Nit, but not necessarily _different_ contracts right?

Comment (shahnami) [reply docs/modules/ROOT/pages/admin.adoc:91]: We clarify that there is a restriction to batches only working for calls on the same network

**Review by shahnami — APPROVED** [review]
LGTM, just a nit pick

File: docs/modules/ROOT/pages/admin.adoc (hunk 1 above)
Comment (ernestognw) [reply docs/modules/ROOT/pages/admin.adoc:91]: Both are right. Not necessarily different contracts, and on the same network.
Let me add those notes. Thanks @shahnami!

**Pull request merged**